wbooks/src/app/screens/booklist/components/BookItem/index.js

```diff
@@ -1,21 +1,30 @@
 import React, { Component } from 'react';
 import { View, Image, Text } from 'react-native';
 import styles from './styles';
+import PropTypes from 'prop-types';
```

Comment:
Since this line is a  **node_modules** import, it should be with the **node_modules** imports.

The order of the imports must be like this (always separated with a blank space):

1) node_modules (like 'react', 'react-native', 'prop-types') and aliases (you will se this later, like '@assets/image.png')
2) like '../CustomBla/index'
3) like './styles'

So in this case you should order the imports in this way:

```
import React, { Component } from 'react';
import { View, Image, Text } from 'react-native';
import PropTypes from 'prop-types';

import styles from './styles';
```

Reply:
ok, I'll keep that in mind